CONTRIBUTING.md

```diff
@@ -1,143 +1,177 @@
+<details>
+  <summary>To use this example as a Git-Hook run the following commands (Click me, I am expandable)</summary>
+    <strong>Consider examining the file before running random code from a public Git repo.</strong>
+
+```shell
+# Ensure the file is executable
+chmod +x .git/hooks/pre-commit
+# Create a link, alternatively use 'cp' instead of 'ln' to copy
+ln ./pre-commit.sh .git/hooks/pre-commit
+# Lets see how long execution takes
+time .git/hooks/pre-commit
 ```
-
-### Quellcode Formatierung
-
-Wir verwenden php-cs-fixer für PHP und ESLint und Prettier für JS um einen gemeinsamen Code Style zu etablieren. Bitte stelle sicher, dass dein Code automatisch richtig formatiert wird, bevor du diesen ins Git repo committest.
-
-Wir empfehlen deine [IDE so zu konfigurieren](https://github.com/ecamp/ecamp3/wiki/installation-development-windows#code-auto-formatting), dass dein Code beim Speichern automatisch richtig formatiert wird.
-
-Alternativ kannst du
-
-- php-cs-fixer und ESLint / Prettier vor jedem commit manuell laufen lassen:
-  ```bash
-  docker compose run api composer cs-fix
-  docker compose run frontend npm run lint
-  docker compose run print npm run lint
-  ```
-- einen git pre-commit hook einrichten, der php-cs-fixer und ESLint vor jedem commit triggert
-
-### Vor dem Einreichen eines Pull Requests
-
-- [x] Wurden alle geänderten oder neuen PHP-Dateien von cs-fixer verarbeitet?
-- [x] Wurden alle geänderten oder neuen JS/Vue-Dateien von ESLint / Prettier bereinigt?
-- [x] Sind alle Variabeln, Klassen, Funktionen, Kommentare, etc. auf englisch benannt?
-- [x] Hast du für neue Funktionalität Tests geschrieben und für geänderte Funktionalität die Tests angepasst?
-- [x] Wurden alle Passwörter, Zugangsdaten und lokale Konfiguration aus den Code-Änderungen entfernt?
-- [x] Enthalten alle geänderten Dateien auch wirklich eine sinnvolle Änderung (im Gegensatz zu z.B. nur Whitespace-Änderungen)?
-- [x] Ist der Fork auf dem aktuellen Stand des zentralen Repositories und können die Änderungen automatisch gemerged werden?
-- [x] Ist der GitHub Actions CI-Build erfolgreich und ohne Test Failures durchgelaufen?
+</details>
+
+#### Checklist :pencil:
+
+We truly value and appreciate every contribution to our project! :heart:
+To make the collaboration smoother and more enjoyable for everyone, 
+we've put together this checklist :scroll:.
+Following it will not only enhance the quality and consistency of your contributions:sparkles:  but also fast-track the review process. :rocket:
+
+
+- [x] **Sync with Central Repository:** :arrows_counterclockwise: Ensure your fork is current with the central repository, facilitating a smooth merge.
+- [x] **Lint:** :wrench: Ensure that linters have run over all new or modified files
+- [x] **Spelling:** :abc: Ensure that linters have run over all new or modified files
```

Comment:
```suggestion
- [x] **Spelling:** :abc: Ensure that the changes are spell checked
```

Reply:
Spell checking is also mentioned below

Reply:
I probably could improve the wording on those. What I meant to express is that the .md files should be spell-checked. Also, the variable names should be spellchecked and the comments should be spell-checked (I don't think it is that important on comments tho. I prefer comments with grammar mistakes to no comments or comments with technical mistakes)